corehq/apps/api/tests/audit_resources.py

```diff
@@ -201,10 +201,47 @@ def test_request_with_date_param(self):
 
         self.assertEqual(result_objects, expected_result_objects)
 
+    def test_request_with_UTC_start_time_start_param(self):
+        start_date = datetime(2023, 5, 2, 1, tzinfo=pytz.timezone('UTC')).isoformat()
```

Comment:
nit: I think a "start date" (similarly with "end date") implies the time would be `0`, otherwise it seems more like a "start time". But I could also see arguments otherwise.

Reply:
Good call [84e76f7](https://github.com/dimagi/commcare-hq/pull/33238/commits/84e76f7c740e4c477640dc6a6db129b97ef9ef16)